src/posts/votes.js

```diff
@@ -258,7 +258,7 @@ module.exports = function (Posts) {
 		const topicData = await topics.getTopicFields(postData.tid, ['mainPid', 'cid', 'pinned']);
 
 		if (postData.uid) {
-			if (postData.votes > 0) {
+			if (postData.votes !== 0) {
```

Comment:
Saving upvotes **and** downvotes to the `:votes` set. Will need an upgrade script to update the zset to include posts with negative vote counts.